src/views/layouts/playGame.hbs

```diff
@@ -1,20 +1,20 @@
 <script src="https://cdn.socket.io/4.3.1/socket.io.min.js"></script>
   <div class="container text-center">
-        <h1>Game Play Page</h1>
+        <h1>Spiel Verwaltung</h1>
```

Comment:
If we switch to german this maybe should be consistent - i would keep the english version but we can discuss that

Reply:
I have chnaged the whole Page to german because maybe this is clearer for the normal german user